app/screens/channel/channel_nav_bar/channel_nav_bar.js

```diff
@@ -35,6 +38,10 @@ export default class ChannelNavBar extends PureComponent {
         theme: PropTypes.object.isRequired,
     };
 
+    static contextTypes = {
```

Comment:
as this component is adding support for `intl` can we make it so it does not need to be re-visited when upgrading to react-intlv3 ? basically use the props approach instead of context.

Reply:
Sure, noted.